src/server.rs

```diff
@@ -894,6 +898,44 @@ impl WalletService for Arc<crate::wallet::Wallet> {
         };
         Ok(tonic::Response::new(response))
     }
+
+    async fn list_transactions(
+        &self,
+        ListTransactionsResponserequest: tonic::Request<ListTransactionsRequest>,
+    ) -> Result<tonic::Response<ListTransactionsResponse>, tonic::Status> {
+        let transactions = self
+            .list_wallet_transactions()
+            .await
+            .map_err(|err| err.into_status())?;
+
+        let response = ListTransactionsResponse {
+            transactions: transactions.into_iter().map(|tx| WalletTransaction {
+                txid: Some(ReverseHex::encode(&tx.txid.to_byte_array())),
```

Comment:
Use `Some(ReverseHex::encode(&tx.txid))` instead

Reply:
If there's a type mismatch from `bitcoin` vs. `bdk::bitcoin`, there's a `convert` module that either contains a function for converting between the types, or where you can add a function that converts